tests/TestCase/Database/Query/SelectQueryTest.php

```diff
@@ -3260,6 +3260,7 @@ public function testDebugInfo(): void
             'params' => [
                 ':c0' => ['value' => '1', 'type' => 'integer', 'placeholder' => 'c0'],
             ],
+            'role' => 'write',
```

Comment:
Nitpick: Even though this is a test assert using `Connection::ROLE_WRITE` instead of a string would be better.

Reply:
Sure, changed.